crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -915,8 +923,16 @@ impl<'db> TypeInferenceBuilder<'db> {
     ///   - `from .foo import bar` => `tail == "foo"`
     ///   - `from ..foo.bar import baz` => `tail == "foo.bar"`
     fn relative_module_name(&self, tail: Option<&str>, level: NonZeroU32) -> Option<ModuleName> {
+        tracing::trace!(
+            "Attempting to resolve module '{}' relative to file '{}'",
+            Self::format_import_from_module(level.get(), tail),
```

Comment:
I would make this a standalone function. Makes it easier to call.

Reply:
I'm not if we want to both log attempting and the resolved module. It seems relatively verbose.